contratospr/contracts/views.py

```diff
@@ -191,6 +197,150 @@ def contractors(request):
     return render(request, "contracts/contractors.html", context)
 
 
+def get_trend(fiscal_year):
+    start_date = timezone.make_aware(datetime.datetime(fiscal_year, 7, 1))
+    end_date = timezone.make_aware(datetime.datetime(fiscal_year + 1, 6, 30))
+    contracts = (
+        Contract.objects.select_related("service__group")
```

Comment:
@froi we might need to add "service".

```suggestion
        Contract.objects.select_related("service", "service__group")
```